### PR TITLE
MB-3639 Make download-secure-migrations script govcloud only

### PR DESCRIPTION
**Pull request by kilbergr**
## Description

https://dp3.atlassian.net/browse/MB-3639

The download-secure-migration script (https://github.com/transcom/mymove/blob/master/scripts/download-secure-migration) pulls secure migrations from the commercial cloud. Now that we've switched over completely to GovCloud, however, this script has been change to accommodate.

* [x] Update docs here: https://github.com/transcom/mymove/wiki/migrate-the-database#downloading-secure-migrations and https://github.com/transcom/mymove/wiki/use-mtls-with-cac

## Reviewer Notes

I pulled out the `run` command. I don't know why (found the same thing in the upload script) but it fails when I use that but works perfectly well without. Please some person more familiar with bash inform me why and how to use! TY

## Setup

Follow instructions here: https://github.com/transcom/mymove/wiki/migrate-the-database#downloading-secure-migrations
BUT use the environments "exp stg prd" if specifying one.

```sh
ENVIRONMENTS=exp download-secure-migration <production_migration_file>
```

## Code Review Verification Steps

  * [x] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [x] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [x] Request review from a member of a different team.
* [x] Have the Jira acceptance criteria been met for this change?

### CODE REVIEW

File: scripts/download-secure-migration [hunk 1]
```diff
@@ -4,14 +4,16 @@
...
 # Environments to download production migrations from
 # shellcheck disable=SC2206
-readonly environments=(${ENVIRONMENTS:-experimental staging prod})
-readonly aws_bucket_prefix="transcom-ppp-app-"
-readonly aws_bucket_suffix="-us-west-2"
+readonly environments=(${ENVIRONMENTS:-exp stg prd})
+readonly aws_gov_prefix="transcom-gov-milmove-"
+readonly aws_gov_region="us-gov-west-1"
+readonly aws_gov_bucket_suffix="-app-${aws_gov_region}"
+readonly aws_gov_id_account="transcom-gov-id"
```
Comment (esacteksab) [inline scripts/download-secure-migration:16]: It took me a minute to understand this is the `aws profile` in `~/.aws/config` is it not? If so, `#comments are free`

File: scripts/download-secure-migration [hunk 2]
```diff
@@ -4,14 +4,16 @@
 # https://github.com/transcom/mymove#secure-migrations
 #
 
-set -eu -o pipefail
+set -eux -o pipefail
```
Comment (gidjin) [inline scripts/download-secure-migration:7]: The `x` pollutes the output, probably should drop it before merging

File: scripts/download-secure-migration [hunk 3]
```diff
@@ -4,14 +4,16 @@
...
 
 readonly aws_command="aws"
 # Environments to download production migrations from
 # shellcheck disable=SC2206
-readonly environments=(${ENVIRONMENTS:-experimental staging prod})
-readonly aws_bucket_prefix="transcom-ppp-app-"
-readonly aws_bucket_suffix="-us-west-2"
+readonly environments=(${ENVIRONMENTS:-exp stg prd})
```
Comment (gidjin) [inline scripts/download-secure-migration:12]: should we validate the passed in environments like the `upload-secure-migration` script? I know we didn't before so I'm fine if you leave it out.

**Review by gidjin — APPROVED** [review]
Script works, changes look good to me aside from the `x` on line 7. But I don't think that should hold up the approval.

File: scripts/download-secure-migration (hunk 3 above)
Comment (kilbergr) [reply scripts/download-secure-migration:12]: Great idea.

File: scripts/download-secure-migration (hunk 2 above)
Comment (kilbergr) [reply scripts/download-secure-migration:7]: Good catch! I was debugging :)

**Review by esacteksab — APPROVED** [review]

**Pull request merged**